Cargo.toml

```diff
@@ -7,9 +7,10 @@ readme = "README.md"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 [features]
-default = []
+default = ["viaduct_client"]
```

Comment:
The tests in 'src/client.rs' currently need a real HTTP client, because the server is being mocked. Changing all the tests to use a fake HTTP client is a non-trivial task and, for this reason, we use the provided `ViaductClient`.

It would be great to only enable the feature for tests, but I could not find a way to do that.

Reply:
Ok, there seems to be a [workaround](https://github.com/rust-lang/cargo/issues/2911#issuecomment-749580481) to make this work :)